src/lib/ProductExporter.ts

```diff
@@ -366,9 +398,26 @@ export class ProductExporter {
             });
         });}
     }
+    private async retrieveReferenceCharges(conn: Connection, charges: String[]){
+        let referenceChargesIds =  new Set<string>();
+        
+        charges.filter(charge => charge['enxCPQ__Charge_Reference__r'])
+               .forEach(charge => {referenceChargesIds.add(charge['enxCPQ__Charge_Reference__r']['enxCPQ__TECH_External_Id__c']) });
```

Comment:
And here. I am marking every place where I find it just not to overlook it if you agree with me and want to change it